lib/Fencer/Types.hs

```diff
@@ -138,7 +138,7 @@ instance FromJSON RateLimit where
 -- Corresponds to one YAML file.
 data DomainDefinition = DomainDefinition
     { domainDefinitionId :: !DomainId
-    , domainDefinitionDescriptors :: !(NE.NonEmpty DescriptorDefinition)
+    , domainDefinitionDescriptors :: !(Maybe (NE.NonEmpty DescriptorDefinition))
```

Comment:
Huh. So lyft/ratelimit accepts a missing `descriptors` but not an empty `descriptors: []`?

Reply:
It does accept empty descriptors. Should I change the type constructor from `NE.NonEmpty` to just `[]`?

Reply:
Yep, and then `Maybe` is unnecessary as well.

Reply:
Fixed.